twilight-http/src/request/application/role_connections/mod.rs

```diff
@@ -0,0 +1,4 @@
+mod get_metadata;
+mod set_metadata;
+
+pub use self::{get_metadata::GetMetadata, set_metadata::SetMetadata};
```

Comment:
Naming could be improved, `GetRoleConnection` and `UpdateRoleConnection` sounds a bit better and more futureproof to me.